### DISCUSSION TITLE
The Daemon.

### DISCUSSION THREAD

**Discussion by theironrobin** [theironrobin/siglo#73] [Q&A] [answered]
A daemon is needed if we want siglo to run in the background to collect things like notifications.

I used systemd the first time I created a siglo daemon, `siglo -d` but a lot of pinephone distros do not use systemd (e.g., postmarketOS)

Should we go ahead with using SystemD and forget the other distros or is there a more general approach that would make the maximum number of users happy?

**Comment by Avamander** [comment, marked as answer]
You can't support them all, especially the ones you don't use.

You can also ship multiple solutions, one of them serving the 90% of users and others that serve parts of the rest.

**Reply by theironrobin** [reply]
systemd it is.
